crates/bitwarden-encoding/src/uniffi_support.rs

```diff
@@ -0,0 +1,15 @@
+use crate::{b64::NotB64Encoded, b64url::NotB64UrlEncoded, B64Url, B64};
+
+uniffi::custom_type!(B64, String, {
+    try_lift: |val| {
+        B64::try_from(val.as_str()).map_err(|e: NotB64Encoded| e.into())
```

Comment:
Note that these conversions currently crash the ios app when they fail until #373 is fixed